src/firewall/core/io/zone.py

```diff
@@ -835,6 +848,13 @@ def zone_writer(zone, path=None):
         handler.ignorableWhitespace("  ")
         handler.simpleElement("masquerade", { })
         handler.ignorableWhitespace("\n")
+ 
+    # tcp-mss-clamp
+    if zone.tcp_mss_clamp:
+        attrs = { }
+        handler.ignorableWhitespace("  ")
+        handler.simpleElement("tcp-mss-clamp:", {"value": value })
+        handler.ignorableWhitespace("\n")
```

Comment:
Similar comment as above. This only applies to rich rules so it needs to be moved down inside the `# rules` section.